addons/ai/functions/fnc_rf_returnFire.sqf

```diff
@@ -0,0 +1,33 @@
+ 
+/*
+    Author:
+        Adam Bridgford, Tim Beswick
+
+    Description:
+        Makes unit fire 20 rounds fast.
+        
+    Parameter(s):
+        0: Unit hit <OBJECT>
+        1: Unit causing the damage <SCALAR>
+        
+    Return Value:
+        Nothing
+*/
+ 
+ params ["_unit","_instigator"];
+ 
```

Comment:
You could probably merge rf_aliveCheck into this function if all aliveCheck is doing is checking if the unit is alive.